Content/default/Build.fs

```diff
@@ -23,14 +23,14 @@ Target.create "Clean" (fun _ ->
 Target.create "InstallClient" (fun _ -> run npm [ "install" ] ".")
 
 Target.create "Bundle" (fun _ ->
-    [ "server", dotnet [ "publish"; "-c"; "Release"; "-o"; $"\"{deployPath}\"" ] serverPath
+    [ "server", dotnet [ "publish"; "-c"; "Release"; "-o"; deployPath ] serverPath
```

Comment:
Please could you confirm or verify that this still works if the deploy path has spaces in it? Thanks.

Reply:
![Screenshot 2023-09-29 at 10 44 45](https://github.com/SAFE-Stack/SAFE-template/assets/25330805/4b20fe3e-eecb-45b9-92c5-471ee4b4f516)

Yes this has been checked as per the above - the path for the command has been wrapped in quotes allowing for spaces